contribute/dev_contribution.rst

```diff
@@ -76,25 +82,30 @@ It may be necessary to refactor the current design to enable the easiest possibl
 
 For simple or obvious bugs which have fallen through the testing gaps just the implementation is fine.
 
-GitHub Pull Request
-===================
+Commiting changes to your feature branch
+========================================
 
-Once some changes have been made and local commits committed push your changes to your GitHub OpenCMISS repository (refer to :numref:`Figure %s <OpenCMISS-branching>`).  From there create a pull request from your topic branch to the :term:`prime repository` develop branch.  When creating the pull request make sure to add in the comment ``addresses issue #46`` (of course replace the number 46 with the actual number of the issue you are addressing), or something to that effect.  This will create a link between the issue and the pull request enabling other people to see that you are working on this issue and comment on your work.
+Once some changes have been made and local commits committed push your changes to your GitHub OpenCMISS repository (refer to :numref:`Figure %s <OpenCMISS-branching>`).
 
 The following Git command line commands show an example of how to add all files, commit the changes and push them to a GitHub repository for the first time::
 
    git add .
    git commit -m "Appropiately descriptive message about the changes made."
-   git push -u origin issue46
+   git push -u origin 46_hypothetical_bug
+
+The ``git add`` command will stage the changes you have selected. You can use this command multiple times to add all the necessary files that you intend to commit. The ``git commit`` command will commit these changes with the commit message indicated by the ``-m`` argument. The ``git push`` command will push the changes (and also create the branch if it does not already exist) to your origin repository. Your origin repository represents the OpenCMISS repository you have forked on GitHub.
+
+GitHub Pull Request
+===================
 
-The ``git add`` and ``git commit`` commands should be obvious,  the ``git push`` command sets the local branch ``issue46`` to be linked with the remote branch ``issue46`` in the origin (the default shorthand for your libCellML repository on GitHub) repository, this branch will be created in the origin repository if it doesn't already exist.
+From there create a pull request from your feature branch to the :term:`prime repository` develop branch. When creating the pull request make sure to add in the comment ``addresses issue #46`` (replace the number 46 with the actual number of the issue you are addressing), or something to that effect. This will create a link between the issue and the pull request enabling other people to see that you are working on this issue and comment on your work.
```

Comment:
Hi Andre, committing to the same branch without mentioning the issue in the commit message would update the pull. However, the issue itself would not get updated - this seems important as people won't know what work has been done if they only browse the issues.